src/components/avatar-rectangle/avatar-rectangle.stories.tsx

```diff
@@ -26,36 +22,23 @@ export default {
   },
 } as Meta<typeof AvatarRectangle>
 
-const exampleImageUrl = 'https://picsum.photos/id/206/100/100'
+type Story = StoryObj<typeof AvatarRectangle>
 
-export const DefaultUsage = {
-  render: ({}) => (
-    <ElAvatarRectangle>
-      <img src={exampleImageUrl} alt="example" />
-    </ElAvatarRectangle>
-  ),
-}
+/**
+ * The default usage use the residential variant and medium size, each can be set using the `data-` attribute.
+ */
```

Comment:
migrate the desc from mdx to here so mdx just need to use `<Description of={stories} />`